artifacts/scw_artifact.py

```diff
@@ -0,0 +1,40 @@
+import json
+import requests
+import tests.utils
```

Comment:
Now some of these test utils are used to build an artifacts, would it make sense to move them to out from `tests` ?

Reply:
yes!  thoughts on a good place to put them?

Reply:
maybe a new dir called `utils` or maybe all code (non json taxonomy/schema stuff) should go under `lib`

Reply:
Unsure of what the convention is with `python` but `utils` sounds like a good name.

Reply:
updated file locations.  I put everything under lib so it doesn't look so messy and segregates the main taxonomy/schema from dev concerns.